Framework/DataObjects/src/PeaksWorkspace.cpp

```diff
@@ -149,6 +150,28 @@ void PeaksWorkspace::removePeak(const int peakNum) {
   peaks.erase(peaks.begin() + peakNum);
 }
 
+/** Removes multiple peaks
+* @param badPeaks peaks to be removed
+*/
+void PeaksWorkspace::removePeaks(std::vector<int> &badPeaks) {
```

Comment:
I would slightly disagree here. `removePeaks(std::vector<int> &&badPeaks)` forces every caller to use a `move`, even when that might not be what they want to do - forcing them to copy the badPeaks list first if they want to preserve it for some reason.

How about `removePeaks(std::vector<int> badPeaks)` taking the vector by value? If you keep your call as `removePeaks(std::move(badPeaks))` then it will still just move the original vector but in the case where the list wants to be preserved the caller doesn't move and the compiler copies the vector (as the user would have had to do anyway).